IBPSA/BoundaryConditions/SolarIrradiation/BaseClasses/RelativeAirMass.mo

```diff
@@ -1,6 +1,7 @@
 within IBPSA.BoundaryConditions.SolarIrradiation.BaseClasses;
 block RelativeAirMass "Relative air mass"
   extends Modelica.Blocks.Icons.Block;
+  parameter Modelica.SIunits.Length alt = 1 "Location altitude above sea level";
```

Comment:
@EttoreZ : shouldn't we just set this to zero as the default?

Reply:
It makes more sense to use zero as a default for sure. Furthermore, that parameter disappeares in the #1518 pull request and becomes a real input (read from weaBus).